src/Microsoft.Azure.NotificationHubs/XiaomiNotification.cs

```diff
@@ -0,0 +1,74 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License. See License.txt in the project root for 
+// license information.
+//----------------------------------------------------------------
+
+using System;
+using System.Text;
+
+namespace Microsoft.Azure.NotificationHubs
+{
+    /// <summary>
+    /// Represents a Xiaomi notification.
+    /// </summary>
+    public sealed class XiaomiNotification : Notification, INativeNotification
+    {
+        static string contentType = "application/json";
```

Comment:
Should use constants instead of static strings when possible, using `PascalCase` instead of `camelCase`